src/sage/interfaces/regina.py

```diff
@@ -0,0 +1,1048 @@
+        r"""
+        Perform a function call.
+
+        EXAMPLES::
+
+            sage: regina._function_call(regina.Polynomial, (-3, 5/3))
+            <regina.Polynomial: 5/3 x - 3>
+        """
+        args, kwds = self._convert_args_kwds(*args, **kwds)
+        if len(args) == 0:
+            if len(kwds) == 0:
+                res = name()
+            else:
+                res = name(**kwds)
+        elif len(args) == 1:
+            if len(kwds) == 0:
+                res = name(args[0])
+            else:
+                res = name(args[0], **kwds)
+        else:
+            if len(kwds) == 0:
+                res = name(*args)
+            else:
+                res = name(*args, **kwds)
+
+        # read back new values of the arguments and keywords
+        def read_back(arg):
+            if isinstance(arg, self._object_class()):
+                self.set(arg._name, arg._inst)
+        for arg in args:
+            read_back(arg)
+        for val in kwds.values():
+            read_back(val)
+        if res is not None:
+            if self._regina_object(res):
+                new = res.__class__(res)  # this is the way to get a copy of a Regina object
+                return self(new)
+            else:
+                return res
+
+    def _equality_symbol(self):
+        r"""
+        EXAMPLES::
+
+            sage: regina._equality_symbol()
+            '=='
+        """
+        return '=='
+
+    def _object_class(self):
+        r"""
+        Return the element class of this parent.
+        This is used in the interface class.
+
+        EXAMPLES::
+
+            sage: regina._object_class()
+            <class 'sage.interfaces.regina.ReginaElement'>
+        """
+        return ReginaElement
+
+    def help(self, cmd, long=False):
+        r"""
+        Return the Regina documentation of the given command.
+
+        EXAMPLES::
+
+            sage: regina.help('AbelianGroup')
+            Represents a finitely generated abelian group.
+            <BLANKLINE>
+            The torsion elements of the group are stored in terms of their
+            invariant factors. For instance, Z_2+Z_3 will appear as Z_6, and
+            Z_2+Z_2+Z_3 will appear as Z_2+Z_6.
+            <BLANKLINE>
+            In general the factors will appear as Z_*d0*+...+Z_*dn*, where the
+            invariant factors *di* are all greater than 1 and satisfy
+            *d0*|*d1*|...|*dn*. Note that this representation is unique.
+            <BLANKLINE>
+            This class implements C++ move semantics and adheres to the C++
+            Swappable requirement. It is designed to avoid deep copies wherever
+            possible, even when passing or returning objects by value.
+        """
+        self._lazy_init()
+        dic = self._namespace.__dict__
+        if cmd in dic:
+            return print(dic[cmd].__doc__)
+        raise NotImplementedError('no documentation available for %s!' % cmd)
```

Comment:
```suggestion
        raise NotImplementedError('no documentation available for %s' % cmd)
```
No need to be excited about it.